### DISCUSSION TITLE
空きスペースにユニバーサル部分を設けてはどうでしょか？

### DISCUSSION THREAD

**Discussion by tatsu1225** [algyan/XIAOGYAN#16] [Ideas]
こちらに関連する話です
https://github.com/algyan/algyan_8th/discussions/13

オンボードにデバイスを追加したい要求は地味にありそうなので，(空きスペースがあれば) ユニバーサル基板としてのスルーホールを設けるのはいかがでしょうか？

最小サイズとしては，GROVEコネクタ近くに2.54ピッチで1*4(VCC, GND, D4/SDA, D5/SCL)なんてどうかな？と思いました :-)

**Comment by matsujirushi** [comment]
Groveコネクタ近くに、2.54mmピッチで4つのスルーホール（Vcc, GND, SDA, SCL）があると、オンボードにセンサーなどを追加しやすいという意味でしょうか？
それとも、ユニバーサル基板風に2.54mmピッチで、例えば10x10のスルーホールを用意したほうが良いという意味？？

**Reply by tatsu1225** [reply]
理想は後者(例えば10✕10)で、スペース的に難しければ前者(4つのスルーホール)の意図でしたー！！

**Reply by matsujirushi** [reply]
なるほど納得です！
空きスペースがあれば、ユニバーサル基板風を設けましょう。

空きスペースが無いときに、2.54mmピッチで4つのスルーホール（Vcc, GND, SDA, SCL）を用意したほうが良いですか？
（これってGroveコネクタなので、そこから配線してもらえばいいんじゃないかという気がします。）

**Reply by tatsu1225** [reply]
> 空きスペースがあれば、ユニバーサル基板風を設けましょう。

ありがとうございます :-) ぜひにー！！

>空きスペースが無いときに、2.54mmピッチで4つのスルーホール（Vcc, GND, SDA, SCL）を用意したほうが良いですか？
>（これってGroveコネクタなので、そこから配線してもらえばいいんじゃないかという気がします。）

・GROVEコネクタは実装しておきたい一方で，GROVEコネクタへのランドに線材を配線するのはちょっとな．
・オンボード化するなら表面は部品でいっぱいなので，基板裏に追加する基板orモジュールを取り付けたいであろう
という点から，2.54mmピッチでVCC, GND, SDA, SCLのスルーホールも置いておいたら，ピンヘッダでも線材でも拡張できそうなのでどうかな．と思いました．あんまりですかねー？

**Comment by matsujirushi** [comment]
@tatsu1225 さん、

ユニバーサル用意しました。 ( https://github.com/algyan/algyan_8th/discussions/18#discussioncomment-4618862 )
どうでしょう？

**Reply by tatsu1225** [reply]
見ました！groveモジュール風としてパージできる形にしたんですね。最高です！
